### PR TITLE
Tweaks to vaccine derivation language/flow

### PR DESCRIPTION
**Pull request by catherio**
Rewriting some things in a way that makes more sense to my brain

### CODE REVIEW

File: src/posts/paper/14-research-sources.ts [hunk 1]
```diff
@@ -330,11 +333,10 @@ Data from [Moderna’s phase 3 study](https://www.nejm.org/doi/full/10.1056/NEJM
...
+Fortunately, we have access to a [supplement provided to the FDA](https://www.fda.gov/media/144453/download), which provides a useful clue about reduction of non-symptomatic cases. In this supplementary data, Moderna showed a 63% reduction in positive PCR swabs at the time of the 2nd dose among people who had no symptoms of COVID up to that point. Some of these people might've later shown symptoms. This indicates that the total reduction in Later-Symptomatic + Never Symptomatic COVID cases from *the first dose* of Moderna’s vaccine was 63%.
 
-This indicates that the total reduction in Symptomatic + Never Symptomatic COVID cases from *the first dose* of Moderna’s vaccine is 63%. Likely it is even better after the second dose. From this, we construct the following chart:
+We therefore use \`1 - 0.63 = 0.37 \` as a conservative estimate of the reduction of total COVID cases in the vaccinated group. Likely it is even better after the second dose. From this, we construct the following chart:
```
Comment (beshaya) [inline src/posts/paper/14-research-sources.ts:339]: should we mention that there are considerable error bars on using data from a single day snapshot like this?

**Review by beshaya — APPROVED** [review]

**Comment by catherio** [comment]
Whoops, forgot I still had to explain that 1.2x figure. Happy for this to go in now.

**Comment by catherio** [comment]
wait a sec @beshaya, now that I wrote it out, it doesn't accord with the table. Right? 37 / (222 + 37) is 14% but it's supposed to be 17% if we're using Byambsuren.

**Comment by blanchardjeremy** [comment]
@beshaya @catherio  - I'm doing a little housekeeping. Do y'all want to keep this one open?

**Comment by beshaya** [comment]
Closing this since it is superseded by Delta re-write. I did take cues from these suggestions in the rewrite!

**Pull request closed without merging**